changelog/@unreleased/pr-684.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: add caching to git describe
```

Comment:
We need a better description here. I'd make it a `break` and say it removes support for Gradle 5 (and say what is the new minimum version we support). I'd also a bit more detail, saying how the pattern of calling `gitversion()` in each project used to lead to extremely long configuration times, however these are now cached so it's faster.